.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+#### Provide a general summary of your changes in the Title above.
+
+## **Description**
+#### _Describe your changes in detail_
+
+
+#### _This project only accepts pull requests related to open issues._
+#### _If suggesting a new feature or change, please discuss it in an issue first._
+
+
+## **Motivation** 
+#### _Why is this change required? What issue does it resolve?_
+
+
+## **Test** **Conditions**
+#### _Describe in detail how the changes are tested._ 
+#### _Include details of your testing environment, and the tests you ran to._
+#### _How does your change affect the rest of the code._ 
+
+
+## **Types of changes**
+#### Check boxes that apply:
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
```

Comment:
```suggestion
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update
```